src/components/v2/shared/FundingCycleConfigurationDrawers/NFTDrawer/MaxSupplyFormItem.tsx

```diff
@@ -0,0 +1,50 @@
+import { t } from '@lingui/macro'
+import { Form, FormInstance } from 'antd'
+import InputAccessoryButton from 'components/shared/InputAccessoryButton'
+import FormattedNumberInput from 'components/shared/inputs/FormattedNumberInput'
+import TooltipLabel from 'components/shared/TooltipLabel'
+
+import { NFTFormFields } from './NFTRewardTierModal'
+
+export default function MaxSupplyFormItem({
+  form,
+}: {
+  form: FormInstance<NFTFormFields>
+}) {
+  const validateMaxSupply = () => {
+    const value = form.getFieldValue('maxSupply')
+    if (value === undefined || value <= 0) {
+      return Promise.reject(t`Maximum supply required`)
+    }
+    return Promise.resolve()
+  }
+
+  return (
+    <div style={{ display: 'flex' }}>
```

Comment:
**question** do we need this wrapping div?

Naming the component `MaxSupplyFormItem` implies that we're export a `Form.Item`, when we're actually exporting a `div`.

We should either:
- remove this div entirely
- Remove it from this component, and wrap the `MaxSupplyFormItem` instance in this `div`